provision/rosa-cross-dc/Taskfile.yml

```diff
@@ -0,0 +1,314 @@
+# https://taskfile.dev
+
+version: '3'
+
+output: prefixed
+
+vars:
+  # To compute Infinispan namespace
+  KC_NAMESPACE_PREFIX: '{{default "$(whoami)-" .KC_NAMESPACE_PREFIX}}'
+  KC_ISPN_NAMESPACE: '{{.KC_NAMESPACE_PREFIX}}keycloak'
+  ISPN_DIR: "../infinispan"
+  RDS_DIR: "../aws/rds"
+  KC_DIR: "../openshift"
+  RS_HOT_ROD_PASSWORD:
+    sh: aws secretsmanager get-secret-value --region eu-central-1 --secret-id keycloak-master-password --query SecretString --output text --no-cli-pager
+
+dotenv: [ '.env' ]
+
+includes:
+  common:
+    taskfile: ../common
+    internal: true
+  ispn:
+    taskfile: ../infinispan/Utils.yaml
+    dir: ../infinispan/
+    internal: true
+
+tasks:
+
+  create-aurora:
+    internal: true
+    desc: "Creates a Aurora Database"
+    requires:
+      vars:
+        - AURORA_CLUSTER
+        - AURORA_REGION
+    env:
+      AURORA_CLUSTER: "{{.AURORA_CLUSTER}}"
+      AURORA_REGION: "{{.AURORA_REGION}}"
+    cmds:
+      - echo "Deploying Aurora Database ({{.AURORA_CLUSTER}}) in region {{.AURORA_REGION}}"
+      - ../aws/rds/aurora_create.sh
+
+  create-peering-connection:
+    desc: "Creates Aurora Peering Connection"
+    internal: true
+    silent: false
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME
+        - AURORA_CLUSTER
+        - AURORA_REGION
+    cmd: CLUSTER_NAME="{{.ROSA_CLUSTER_NAME}}" KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}" SKIP_ROSA_LOGIN=true {{.RDS_DIR}}/aurora_create_peering_connection.sh
+    preconditions:
+      - test -f {{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}
+
+  fetch-aurora-endpoint:
+    internal: true
+    requires:
+      vars:
+        - AURORA_CLUSTER
+    cmd: ../aws/rds/aurora_endpoint.sh > .task/aurora-endpoint-{{.AURORA_CLUSTER}}
+    generates:
+      - .task/aurora-endpoint-{{.AURORA_CLUSTER}}
+    status:
+      - ! test -f .task/aurora-endpoint-{{.AURORA_CLUSTER}}
+
+  deploy-infinispan-crossdc:
+    desc: "Deploys Infinispan in both ROSA clusters"
+    internal: true
+    cmds:
+      - echo "Deploying Infinispan cluster with cross-site in ROSA clusters {{.ROSA_CLUSTER_NAME_1}} and {{.ROSA_CLUSTER_NAME_2}} (namespace={{.KC_ISPN_NAMESPACE}})"
+      - task: ispn:crossdc
+        vars:
+          OC_NAMESPACE_1: "{{.KC_ISPN_NAMESPACE}}"
+          OC_NAMESPACE_2: "{{.KC_ISPN_NAMESPACE}}"
+          CROSS_DC_HOT_ROD_PASSWORD: "{{.RS_HOT_ROD_PASSWORD}}"
+
+
+  fetch-ocp-hostname-prefix:
```

Comment:
this task will become redundant after https://github.com/keycloak/keycloak-benchmark/pull/484/files#diff-246cc39ce6408ddc2fe1cf751d9d66f9d87c95fc82ea8969990490e51ba19625R24 is merged.